src/main/blackbox/blackbox.c

```diff
@@ -1384,15 +1384,17 @@ static bool blackboxWriteSysinfo(void)
 #ifdef USE_INTEGRATED_YAW_CONTROL
         BLACKBOX_PRINT_HEADER_LINE("use_integrated_yaw", "%d",              currentPidProfile->use_integrated_yaw);
 #endif
-        BLACKBOX_PRINT_HEADER_LINE("feedforward_transition", "%d",          currentPidProfile->feedForwardTransition);
-        BLACKBOX_PRINT_HEADER_LINE("feedforward_weight", "%d,%d,%d",        currentPidProfile->pid[PID_ROLL].F,
+        BLACKBOX_PRINT_HEADER_LINE("ff_transition", "%d",                   currentPidProfile->feedforwardTransition);
+        BLACKBOX_PRINT_HEADER_LINE("ff_weight", "%d,%d,%d",                 currentPidProfile->pid[PID_ROLL].F,
```

Comment:
I am  a bit split on renaming entries in the blackbox header just for renaming's sake - this will break all existing software that will try to read / use this value.
On the other hand it can be said that all other changes to the blackbox header in this pull request make it unlikely that software will be breaking only because of this.

Reply:
@mikeller I also was a bit torn on the virtue of being 'consistent' in the log viewer for feedforward.  
`ff` is certainly more compact than `feedforward`.  However, when searching the code for variables and values related to a function, its a lot easier if we use the same nomenclature everywhere.  It was that 'principle' that swayed to me change to `feedforward` in the logging code.  Hence I am reluctant to make it that only here in blackbox, and nowhere else, would we use `ff`.  
And, as you point out, there will need to be version-related changes for rc smoothing names, so making changes for feedforward will follow that pattern.  Hence it seems best to make all the name changes at once; ie, that now seems a good time to do the painful, but probably correct, thing, I guess.